.github/workflows/companion-deploy.yml

```diff
@@ -43,10 +44,13 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+      - name: Alter dockerfile
+        run: |
+          sed -i 's/3020/$PORT/g' Dockerfile
```

Comment:
```suggestion
          sed -i 's/^EXPOSE 3020$/EXPOSE $PORT/g' Dockerfile
```

Reply:
I think matching the whole line is safer, no? Other than that it looks good to me!

Reply:
Makes sense yes, committed your change